charts/warpstream-agent/templates/deployment.yaml

```diff
@@ -119,9 +119,13 @@ spec:
             failureThreshold: 5
             periodSeconds: 5
           resources:
+            {{- $goMaxProcs := 0 -}}
```

Comment:
Have to define the variable outside the scope of the if/end.  0 is arbitrary choice.